components/KeyManagerNonceChecker/KeyManagerNonceChecker.tsx

```diff
@@ -51,6 +53,10 @@ const KeyManagerNonceChecker: React.FC = () => {
   return (
     <div className="container mt-5">
       <h2 className="title is-2">Nonce</h2>
+      <div className="tags has-addons">
```

Comment:
Suggest removing this, as we already have the network toggle at the top right when selecting it